packages/webapp/src/routes/index.jsx

```diff
@@ -901,7 +899,6 @@ const Routes = ({ isCompactSideMenu }) => {
             />
             <Route path="/finances" component={Finances} />
```

Comment:
@antsgar Just noticed.. is this missing a `/finances/* `like the other one in this file?